### PR TITLE
576 layout of multiple visualization

### PR DESCRIPTION
**Pull request by longshuicy**
This PR:

1. use grid layout for multiple visualization component
2. simplify the logic of render vis on each component (not using redux anymore)
image

### CODE REVIEW

**Comment by sandeep-ps** [comment]
Please resolve the merge conflicts. Thanks.

File: backend/app/routers/visualization.py [hunk 1]
```diff
@@ -172,23 +171,29 @@ async def save_visualization_config(
...
     visconfigs = []
     async for vzconfig in VisualizationConfigDB.find(*query):
-        visconfigs.append(vzconfig)
+        config_visdata = []
+        visdata_query = [VisualizationDataDB.visualization_config_id == vzconfig.id]
+        async for vis_data in VisualizationDataDB.find(*visdata_query):
+            config_visdata.append(vis_data.dict())
+        visconfig_out = VisualizationConfigOut(**vzconfig.dict())
+        visconfig_out.visualization_data = config_visdata
+        visconfigs.append(visconfig_out)
```
Comment (longshuicy) [inline backend/app/routers/visualization.py:186]: The attaching visdata model in config needed for this endpoint. This i the endpoint being used by frontend

File: backend/app/models/visualization_config.py [hunk 2]
```diff
@@ -15,6 +14,7 @@ class VisualizationConfigBase(BaseModel):
     job: Optional[EventListenerJobDB]
     client: Optional[str]
     parameters: dict = {}
+    visualization_component_id: str
```
Comment (ddey2) [inline backend/app/models/visualization_config.py:17]: same member variable as in line 19

**Review by tcnichol — APPROVED** [review]
Tested multiple visualizations tile properly marking approved.

**Review by max-zilla — APPROVED** [review]

**Pull request merged**